### PR TITLE
Remove unused imagePublisher

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Works with refactor branch of `rgbd`

Merge together with tue-robotics/rgbd#26

### CODE REVIEW

**Comment by jlunenburg** [comment]
Do you know what the intended use of this thing was? In any way, good to remove deprecated code ;-)

**Comment by MatthijsBurgh** [comment]
> Do you know what the intended use of this thing was? In any way, good to remove deprecated code ;-)

I have no idea, commit history is very short

**Comment by MatthijsBurgh** [comment]
This needs to be merged together with tue-robotics/rgbd#26

File: src/models/shape_loader.cpp [hunk 1]
```diff
@@ -636,6 +636,10 @@ bool readPose(tue::config::Reader& cfg, geo::Pose3D& pose, tue::config::Required
         else
             return false;
     }
+    else
```
Comment (jlunenburg) [inline src/models/shape_loader.cpp:639]: I don't know where the changes in this file come from and how they're related to https://github.com/tue-robotics/rgbd/pull/26/files

Comment (MatthijsBurgh) [reply src/models/shape_loader.cpp:639]: These changes shouldn't be here. I will revert those.

**Comment by reinzor** [comment]
Don't know where the Image publisher was for.

**Pull request merged**